internal/gcsutil/util.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package gcsutil contains utility functions for Google Cloud Storage data access.
+package gcsutil
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+)
+
+// GCSClient contains a Google Cloud Storage client.
+type GCSClient struct {
+	Client *storage.Client
+}
+
+// NewClient creates and returns a new GCSClient.
+func NewClient(ctx context.Context) (*GCSClient, error) {
```

Comment:
maybe `NewGCSClient`?

Reply:
Thanks for the suggestion ! I changed the naming.

Reply:
I wonder if this is not considered a repetition in naming and if it is better to rename the `GCSClient` to `Client` instead, since they are all in the `gcsutil` package.

Reply:
That is a good point. Since the package name is `gcsutil` I think you can drop `GCS`.